source/shared/cpp/ObjectModel/MarkDownParser.cpp

```diff
@@ -0,0 +1,33 @@
+#include <vector>
+#include <iomanip>
+#include <iostream>
+#include "MarkDownParser.h"
+
+using namespace AdaptiveCards;
+
+MarkDownParser::MarkDownParser(const std::string &txt) : m_text(txt)
+{
+}
+
+// transforms string to html
+std::string MarkDownParser::TransformToHtml()
+{
+    ParseBlock();
+
+    m_parsedResult.Translate();
+
+    return "<p>" + m_parsedResult.GenerateHtmlString() + "</p>";
+}
+
+// MarkDown is consisted of Blocks, this methods parses blocks
+void MarkDownParser::ParseBlock() // rename it to tokenize() // state machines
```

Comment:
>rename it to tokenize() [](start = 37, length = 23)

Are you still planning to make this change?

Reply:
i made it tokenize, but it was changed to something else completely. will change the comments; thanks

---
In reply to: [157569340](https://github.com/Microsoft/AdaptiveCards/pull/1045#discussion_r157569340) [](ancestors = 157569340)